src/prometheus_alert_rules/hydra_missing.rule

```diff
@@ -0,0 +1,8 @@
+alert: HydraMissing
+expr: up == 0
+for: 1m
+labels:
+  severity: critical
+annotations:
+  summary: Prometheus target missing (instance {{ $labels.instance }})
+  description: "Hydra target has disappeared. An exporter might be crashed.\n  VALUE = {{ $value }}\n  LABELS = {{ $labels }}"
```

Comment:
I am a little bit confused about this rule with the `HydraUnitDown`. Do they both infer to hydra is not healthy with the only difference in healthy unit number? If so, I guess we could just provide one rule to cover, e.g. the ratio of healthy units is below a certain threshold for duration x.

Reply:
comparison number should be the amount of hydra pods running, also this will allegedly fire for any component monitored by prometheus